src/services/passwordRecovery/hooks/index.js

```diff
@@ -18,6 +21,32 @@ const hashId = (hook) => {
 	}
 };
 
+const sendInfo = (hook) => {
+	if (hook.path === 'passwordRecovery') {
+		hook.app.service('/accounts').get(hook.data.account, { $populate: ['userId'] })
+			.then((account) => {
+				const recoveryLink = `${process.env.HOST}/pwrecovery/${hook.result._id}`;
+				const mailContent = `Sehr geehrte/r " ${account.userId.firstName} ${account.userId.lastName}, \n\n
+					Bitte setzen Sie Ihr Passwort unter folgendem Link zurück:\n
+					${recoveryLink}\n\n
+					Mit Freundlichen Grüßen\n
+					Ihr ${process.env.SC_SHORT_TITLE || 'Schul-Cloud'} Team`;
```

Comment:
duplication L33 und L36 und wahrscheinlich an anderer Stelle schon gemacht (APP)

Reply:
It's done very often like this. Normally in the valediction it is just written Schul-Cloud and no instance specific version. I think this is nothing which should be part of this issue. It need to be solved in the whole project. New ticket?

Reply:
new ticket :) as bug pls